interactive_engine/executor/ir/runtime/src/process/operator/map/auxilia.rs

```diff
@@ -75,7 +75,10 @@ impl FilterMapFunction<Record, Record> for AuxiliaOperator {
                         edge.into()
                     })
                 } else {
-                    Err(FnExecError::unexpected_data_error("should be vertex or edge in AuxiliaOperator"))?
+                    Err(FnExecError::unexpected_data_error(&format!(
+                        "Entry should be vertex or edge in AuxiliaOperator: {:?}",
```

Comment:
neither `Vertex` nor `Edge` entry is accessed in `Auxilia` operator, the entry is {:?}, entry